src/containers/SellerProductsContainer.jsx

```diff
@@ -140,23 +203,14 @@ const SellerProductsContainer = () => {
             </div>
           )}
 
-          {networkError && (
-            <div className='loading_div flex justify-center w-[84%] '>
-              Server or Network Error{' '}
-              <FontAwesomeIcon icon='fa-solid fa-exclamation-circle' />
-            </div>
-          )}
-
           {/* Product in my collection */}
           {myCollection?.map((product, index) => {
             return (
-              <div
-                key={product.id}
-                className='sellingProd__card flex  mb-5 '
-                onClick={() => {
-                  return getProduct(index);
-                }}
-              >
+              <div key={product.id} className='sellingProd__card flex  mb-5 '
+              onClick={() => {
+                // document.querySelector('#one').classList.add('bg-black','bg-opacity-50');
```

Comment:
Please remove these commented lines of code.

Reply:
I've resolved the issue